examples/microprofile/openapi-basic/src/main/java/io/helidon/microprofile/examples/openapi/basic/GreetResource.java

```diff
@@ -128,11 +128,11 @@ public JsonObject getMessage(@PathParam("name") String name) {
         description = "Conveys the new greeting prefix to use in building greetings",
         content = @Content(
                     mediaType = "application/json",
-                    schema = @Schema(implementation = GreetingMessage.class),
+                    schema = @Schema(implementation = GreetingUpdateMessage.class),
                     examples = @ExampleObject(
                         name = "greeting",
                         summary = "Example greeting message to update",
-                        value = "New greeting message")))
+                        value = "{\"greeting\": \"New greeting message\"")))
```

Comment:
This line needs the same syntax change as noted above to add the closing curly brace.